Frontend/src/router/index.js

```diff
@@ -87,4 +101,55 @@ const router = createRouter({
   },
 });
 
+router.beforeEach(async (to, from, next) => {
+  if (!store.getters["auth/getAuthData"].token) {
+    const accessToken = localStorage.getItem("accessToken");
+    const refreshToken = localStorage.getItem("refreshToken");
+    if (accessToken) {
+      const data = {
+        accessToken: accessToken,
```

Comment:
Ich habe mittlerweile gelernt, dass wenn Key und Value den gleichen Namen besitzen, es aussreichend ist nur einen der beiden zu bennen.
Also:
`data = {
accessToken,
refreshToken
}`

Ich werde das in Zukunft so in meinem Code umsetzen. Aber das ist natürlich Geschmackssache.

Reply:
Jap, das kann man machen.